test/test_all.py

```diff
@@ -117,6 +117,11 @@ def __init__(self, arch, container, address, *args, **kwargs):
                "symbexec.py",
                ]:
     testset += RegressionTest([script], base_dir="ir")
+testset += RegressionTest(["analysis.py"], base_dir="ir",
```

Comment:
You may want to avoid `sum()` (or `reduce()`) and prefer a double comprehension.

```Python
[fname for fnames in (["simp_graph_%02d.dot" % test_nb, "graph_%02d.dot" % test_nb]
       for test_nb in xrange(1, 18)) for fname in fnames]
```